### DISCUSSION TITLE
Are buckets the same as repositories?

### DISCUSSION THREAD

**Discussion by trymeouteh** [ScoopInstaller/Scoop#5463] [Help and Questions] [answered]
Can anyone self host their own bucket for their own software?

Can anyone create a bucket (repo) and allow others to get their software to be hosted on this bucket?

Are buckets just repositories for scoop. There are official buckets (repo) for scoop and you can add others that are out there?

**Comment by rashil2000** [comment, marked as answer]
Yes.

Yes.

Yes.
